css/footer.css

```diff
@@ -0,0 +1,83 @@
+footer {
+  height: 846px;
+  position: relative;
+
+  /* text-align: center; */
```

Comment:
- [ ] Please remove this and any commented code that you may have left behind to maintain your project clean :)